examples/evm/buildImportTx-xchain.ts

```diff
@@ -30,18 +31,32 @@ const cAddressStrings: string[] = cchain.keyChain().getAddressStrings()
 const xChainBlockchainId: string = Defaults.network[networkID].X.blockchainID
 
 const main = async (): Promise<any> => {
+  const baseFeeResponse: string = await cchain.getBaseFee()
+  const baseFee = new BN(parseInt(baseFeeResponse, 16))
+  let fee: BN = baseFee
   const evmUTXOResponse: any = await cchain.getUTXOs(
     cAddressStrings,
     xChainBlockchainId
   )
   const utxoSet: UTXOSet = evmUTXOResponse.utxos
+  let unsignedTx: UnsignedTx = await cchain.buildImportTx(
+    utxoSet,
+    cHexAddress,
+    cAddressStrings,
+    xChainBlockchainId,
+    cAddressStrings,
+    fee
+  )
+  const importCost: number = costImportTx(unsignedTx)
+  fee = baseFee.mul(new BN(importCost))
 
-  const unsignedTx: UnsignedTx = await cchain.buildImportTx(
+  unsignedTx = await cchain.buildImportTx(
```

Comment:
Would it make sense to add this part where we call `costImportTx` into `buildImportTx` so that we don't need to call it twice where it actually gets used?

Reply:
Going to revisit this for a future AvalancheJS release since we're pressed for time on this release and I don't want to introduce any more changes which would cause the wallet to need to refactor the integration.